testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -1668,3 +1668,55 @@ def test_dedent_docstring(text):
     doc = util.dedent_docstring(text)
     for line in doc.splitlines():
         assert line == line.lstrip()
+
+
+class TestCheckBox(object):
```

Comment:
I just copied this over from `lib/test_distances.py`. The code itself remained unchanged.